src/car/distance/DistanceCar.hpp

```diff
@@ -15,9 +15,9 @@ namespace constants
 namespace car
 {
 const unsigned long kDefaultPidFrequency = 80;
-const float kDefaultProportional         = 5.0F;
-const float kDefaultIntegral             = 0.0F;
-const float kDefaultDerivative           = 10.0F;
+const float kDefaultProportional         = 5.0f;
+const float kDefaultIntegral             = 0.0f;
```

Comment:
:warning: **readability-uppercase-literal-suffix** :warning:
floating point literal has suffix 'f', which is not uppercase
```suggestion
const float kDefaultIntegral             = F```